pipeline/src/client.py

```diff
@@ -114,7 +120,14 @@ def main(self, read_env_data=False):
     #######################################################
 
     def create_hive(self):
-        self.flags.create_table()
+        if (self._output_type == "aperture" or self._output_type == "both"):
+            self.flags.create_table()
+
+        #When dealing with csv and hive creation, only data available right away is flags, so we only save flags to csv when "creating" csv-hive
+        if (self._output_type == "csv" or self._output_type == "both"):
+            if not self.flags.write_csv(self._output_path):
+                print("Error saving Flags to csv.")
```

Comment:
Why doesn't the flags table get created if the option is csv?

Reply:
I'm confused. Do we want hive to be created even when output is directed to csv? Originally I had hive created only when output is aperture (since there is no need for it if output is csv)

Reply:
Two options: either you create all tables or you don't create any tables at all. If you want to not create any tables then change service_periods store its table in memory instead of on database.

Reply:
Also like I said above,  if you choose to create all tables maybe don't even provide them the option of csv only and always require them to do either database only or both.

Reply:
Good point